src/sider/frisktilarbeid/FriskmeldingTilArbeidsformidling.tsx

```diff
@@ -1,13 +1,18 @@
-import React, { ReactElement, useState } from "react";
+import React, { useState } from "react";
 import { useVedtakQuery } from "@/data/frisktilarbeid/vedtakQuery";
 import { VedtakResponseDTO } from "@/data/frisktilarbeid/frisktilarbeidTypes";
 import { FerdigbehandletVedtak } from "@/sider/frisktilarbeid/FerdigbehandletVedtak";
 import { VedtakFattet } from "@/sider/frisktilarbeid/VedtakFattet";
-import { FattNyttVedtak } from "@/sider/frisktilarbeid/FattNyttVedtak";
+import { FattVedtakSkjema } from "@/sider/frisktilarbeid/FattVedtakSkjema";
 
-export const FriskmeldingTilArbeidsformidling = (): ReactElement => {
+export function FriskmeldingTilArbeidsformidling() {
   const { data } = useVedtakQuery();
   const [isNyVurderingStarted, setStartNyVurdering] = useState(false);
+  const [isFormSubmitted, setIsFormSubmitted] = useState(false);
+  const setFormSubmitted = () => setIsFormSubmitted(true);
+  const [isOppgaveAvsluttetNow, setIsOppgaveAvsluttetNow] = useState(false);
+  const setOppgaveAvsluttetNow = () => setIsOppgaveAvsluttetNow(true);
```

Comment:
Blir veldig mye state greier her nå. Ikke helt sikker på hvordan vi kan unngå det 🤔 Tar gjerne i mot tips eller forslag🙏

Reply:
Hvis det er for å vise en alert etter en sukessfull mutation så går det kanskje an å heller bruke `NotificationContext`-opplegget vi gjør i aktivitetskrav og 8-4. Kan godt prøve meg på et forslag :+1: